Makefile

```diff
@@ -23,6 +23,9 @@ test: fmt vet
 integ-test:
 	go test -v -test.timeout=5m ./tests/integration/... -coverprofile cover.out
 
+test-gc: fmt vet
```

Comment:
Any reason we would need to separate these tests?

Reply:
I need to add a comment. I didn't want or need to run all the tests and I didn't want to manually keep typing `-args -ginkgo.focus=".*garbage\ collection.*"`. It might be helpful to help additional Makefile targets for running subsets of the tests. I am also fine with removing if you think it is confusing.

Reply:
Actually, as a total newbie to running these tests for the first time last week I would have loved to at least have an example of how to target the test suite so that I wouldn't have to wait through and sift through the logging for all of them.  I think this would be a good add if for nothing else than that purpose.

Reply:
I would like to add similar, targets for other tests. The verbose output is painful at times. I looked before briefly at cut down the output to just failures but I was unsuccessful.

Reply:
>The alternative (while doing development) is to make the test use a F infront of the test

Do I need to remove the `F` if I want to remove the focus? If so, that is not a great general solution.

Reply:
Yeah. It's intended for development time focusing.